opencga-app/app/scripts/azure/arm/solr/solr-setup.sh

```diff
@@ -14,27 +14,63 @@ SOLR_VERSION=$3
 
 DOCKER_NAME=opencga-solr-${SOLR_VERSION}
 
-# Create docker
+
+scanForNewDisks() {
+    # Looks for unpartitioned disks
+    DEVS=($(fdisk -l 2>/dev/null | egrep -o '(/dev/[^:]*):' | egrep -o '([^:]*)'))
+
+    for DEV in "${DEVS[@]}";
+    do
+        echo "Found unpartitioned disk $DEV"
+        isPartitioned ${DEV}
+    done
+}
+
+isPartitioned() {
+    OUTPUT=$(partx -s ${1} 2>&1)
+    if [[ "$OUTPUT" == *"failed to read partition table"* ]]; then
+        # found unpartitioned disk
+        formatAndMountDisk "${DEV}"
+    fi
+}
+
+formatAndMountDisk() {
+    #partitions primary linux partition on new disk
+    echo "Partitioning disk"
+    echo 'type=83' | sfdisk ${1}
+    until blkid ${1}1
+    do
+        echo "Waiting for drive to be partitioned"
+        sleep 2
+    done
+    echo "Formatting Partition"
+    mkfs -t ext4 ${1}1
+    mkdir /datadrive
+    mount -o acl ${1}1 /datadrive
+    fs_uuid=$(blkid -o value -s UUID ${1}1)
+    echo "UUID=${fs_uuid}   /datadrive   ext4   defaults,nofail,acl   1   2" >> /etc/fstab
```

Comment:
Is this the stuff from @bart-jansen s work? I think it would be good to pull this out in to a script we can share rather than duplicate. Sorry!

Reply:
Maybe track a backlog item to fix this up and we can go ahead with this approach for now.

Reply:
Yup, agree, I'm goign to struggle to find much time next week :(.

Reply:
That's fine lets add an issue to fix this up and merge as-is for now